tests/test_integrated_channels/test_moodle/test_client.py

```diff
@@ -101,8 +102,13 @@ def test_delete_content_metadata(self):
 
         client = MoodleAPIClient(self.enterprise_config)
         client._post = unittest.mock.MagicMock(name='_post', return_value=SUCCESSFUL_RESPONSE)  # pylint: disable=protected-access
-        client.get_course_id = unittest.mock.MagicMock(name='_get_course_id')
-        client.get_course_id.return_value = self.moodle_course_id
+        client._get_courses = unittest.mock.MagicMock(name='_get_courses')  # pylint: disable=protected-access
```

Comment:
I would also add a negative test case to cover missing course id from moodle side